ts/screens/onboarding/EmailReadScreen.tsx

```diff
@@ -58,22 +69,32 @@ const styles = StyleSheet.create({
 });
 
 export class EmailReadScreen extends React.PureComponent<Props> {
-  private handleGoBack = () =>
-    Alert.alert(
-      I18n.t("onboarding.alert.title"),
-      I18n.t("onboarding.alert.description"),
-      [
-        {
-          text: I18n.t("global.buttons.cancel"),
-          style: "cancel"
-        },
-        {
-          text: I18n.t("global.buttons.exit"),
-          style: "default",
-          onPress: () => this.props.abortOnboarding()
-        }
-      ]
-    );
+  private contextualHelp = {
```

Comment:
If you prefer, you could move it out of the class scope and make it as a constant
```typescript
const contextualHelp = {
  title: I18n.t("email.insert.help.title"),
  body: () => <Markdown>{I18n.t("email.insert.help.content")}</Markdown>
};
```

Reply:
shifted within https://github.com/teamdigitale/io-app/pull/1363/commits/343a443257f50d0e5b6bd0cf3876c20af441282f.

the same pr introduce also the binding of the functions